app/renderer/js/main.ts

```diff
@@ -757,8 +762,11 @@ class ServerManagerView {
 				},
 				{
 					label: 'Notification settings',
+					enabled: this.isLoggedIn(index),
 					click: () => {
-						this.tabs[this.activeTabIndex].webview.showNotificationSettings();
+						// switch to tab whose icon was right-clicked
+						this.activateTab(index);
```

Comment:
Do we need this?

Reply:
Yeah, because I could try clicking on org X's menu while keeping Y active and then Y's notification settings would pop up, which isn't something I really want. 
I think this offers better UX.